### ISSUE TITLE
Horizon overlay not working with plugin version of IINACT

### ISSUE THREAD

**Issue by The-Snades** [marzent/IINACT#73]
Horizon overlay wont get out of the waiting for encounter state when used with IINACT and Browsingway

![image](https://github.com/marzent/IINACT/assets/33063784/53433b47-9a35-4001-89f4-96e53fe60cb2)

Browsingway Inlay config
![image](https://github.com/marzent/IINACT/assets/33063784/882026b8-4c76-4e8e-9479-214950134ee5)
![image](https://github.com/marzent/IINACT/assets/33063784/9799be40-f780-4324-86d5-4576fca3e8ee)

**Comment by LegendEvent** [comment]
same problem

**Comment by LegendEvent** [comment]
> try adding a / at the end of the URL you copied and pasted unto browsingway

i did it: 
![image](https://github.com/marzent/IINACT/assets/23439490/7054e4ea-92de-4dc1-94b2-643092f17cd8)
But it just displays: 
![image](https://github.com/marzent/IINACT/assets/23439490/f0561fb8-c611-4a55-9e78-f498468abbf9)
I have only IINACT installed

**Comment by LegendEvent** [comment]
> > > try adding a / at the end of the URL you copied and pasted unto browsingway
> > 
> > 
> > i did it: ![image](https://private-user-images.githubusercontent.com/23439490/327969374-7054e4ea-92de-4dc1-94b2-643092f17cd8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ4NDk3OTksIm5iZiI6MTcxNDg0OTQ5OSwicGF0aCI6Ii8yMzQzOTQ5MC8zMjc5NjkzNzQtNzA1NGU0ZWEtOTJkZS00ZGMxLTk0YjItNjQzMDkyZjE3Y2Q4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA0VDE5MDQ1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTUxY2M3YTNjYzE4ZTEzZjgwNmUwMjZhZmQ2ODM2Yjg2MzZjZmNlYWIwZmQ4MGI3YTE0ZWFmZWMxY2RjNTNjOGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.3HbdAyot4_soL_eU0ciIqZcUwTCIw7Vb3mRpoQSgLmA) But it just displays: ![image](https://private-user-images.githubusercontent.com/23439490/327969394-f0561fb8-c611-4a55-9e78-f498468abbf9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ4NDk3OTksIm5iZiI6MTcxNDg0OTQ5OSwicGF0aCI6Ii8yMzQzOTQ5MC8zMjc5NjkzOTQtZjA1NjFmYjgtYzYxMS00YTU1LTllNzgtZjQ5ODQ2OGFiYmY5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA0VDE5MDQ1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNiOGU5ODZlNzVjMTU3NGYyNGY4NGZlNjIwNWEyMTc5ZTZjOGVkMmE2NWFlZDhhOTliZWMxZDVmZTY4Y2FhNmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.gSYFexyx07MeAc_LEg8uqFtonJvBkrMQK0XveWtLk58) I have only IINACT installed
> 
> URL for the websocket part at the end shouldn't have # afaik it's not a valid ip address

You're right!! I copied it directly from the browser. An other thing, all other chars wont be displayed there. I can only see my DPS and from EOS (the fairy xD). Do you know how to fix that?

**Comment by LegendEvent** [comment]
> check your parse filter in iinact from self > party > none. if that does not fix it, then it's a setting on the overlay that you use and beyond the control of iinact

i changed it to party, but even then I'll only show up there. I had a look for the log file but that wont be written either. 
I checked every DPS Meter available in IINACT :c

**Comment by Kumicoo** [comment]
Still not working.

**Comment by marzent** [comment]
This is a duplicate of https://github.com/marzent/IINACT/issues/134
